Sources/LicensePlist/main.swift

```diff
@@ -18,6 +18,8 @@ extension CompletionKind {
 
 // Typename used for usage in help command
 struct LicensePlist: ParsableCommand {
+    static let configuration = CommandConfiguration(version: Consts.version)
```

Comment:
There should be no side effects.

https://github.com/apple/swift-argument-parser/blob/d2930e8fcf9c33162b9fcc1d522bc975e2d4179b/Sources/ArgumentParser/Parsable%20Types/ParsableCommand.swift#L34-L42

https://github.com/apple/swift-argument-parser/blob/d2930e8fcf9c33162b9fcc1d522bc975e2d4179b/Sources/ArgumentParser/Parsable%20Types/CommandConfiguration.swift#L50-L89